versions/1/manifest.spec.yml

```diff
@@ -170,27 +170,26 @@ spec:
               examples:
                 - ">=7.9.0"
     input_groups:
-      description: List of policy groups offered by this package.
+      description: List of input groups offered by this package.
       type: array
       items:
         type: object
         additionalProperties: false
         properties:
           name:
-            description: Name of policy group.
+            description: Name of input group.
             type: string
             examples:
-              - Logs
-              - Metrics
+              - logs
+              - metrics
           title:
-            description: Title of policy group.
+            description: Title of input group.
             type: string
             examples:
               - Logs
               - Metrics
-              - Checks
```

Comment:
I'm not sure why we included this example, as there doesn't seem to be a corresponding `name` or `description` example for it. So I think we should remove it.

Reply:
I think the "examples" field is not an enum. It just gives samples the user can use. It doesn't mean it has to be consistent across examples for other field values.

Reply:
I understand that it just gives sample values. I just think it's confusing if there isn't consistency between this field and the `name` field, especially since the first two examples here line up exactly with the ones in the `name` field. 

Maybe one solution might be to use examples here that are completely different from the `name` values? I think one related question is: how/where do we expect this `title` field to be used?

Reply:
> Maybe one solution might be to use examples here that are completely different from the name values?

It's a good idea, might be less misleading.

>  I think one related question is: how/where do we expect this title field to be used?

I see this as a safe workaround for multi-words names, let's say: magic_checks/multichecks vs "Magic checks".

Reply:
Thanks, I will update the examples to be completely different from the `name` values and include some multi-word ones.